packages/wrapper/src/editorAppBase.ts

```diff
@@ -109,7 +109,7 @@ export abstract class EditorAppBase {
     }
 
     async createEditors(container: HTMLElement): Promise<void> {
-        if (this.getConfig().useDiffEditor) {
+        if (this.getConfig().useDiffEditor === true) {
```

Comment:
nitpicking: it seems what you want here is a default value for the `useDiffEditor` flag, using `=== true` for it is confusing to me because the default value doesn't appear anywhere. I would prefer `if (this.getConfig().useDiffEditor ?? false) {`

Reply:
Good point, `??` syntax is more readable. In the end both statements lead to the same result.

Reply:
Done ⬇️